hieradata/staging.yaml

```diff
@@ -25,6 +25,7 @@ govuk::apps::email_alert_api::govuk_notify_base_url: 'https://api.staging-notify
 govuk::apps::email_alert_api::govuk_notify_template_id: '76d21ce7-54c3-4fb7-8830-ba3b79287985'
 govuk::apps::hmrc_manuals_api::publish_topics: false
 govuk::apps::kibana::logit_environment: d414187a-2796-4ea7-9b9a-d40c341646d6
+govuk::apps::link_checker_api::govuk_basic_auth_credentials: "Basic %{hiera('http_username')}:%{hiera('http_password')}"
```

Comment:
Sorry I've changed my mind... I think we shouldn't include the `Basic` in the string here as the variable has `basic` in the name.

Reply:
🙃